specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/connectionsCreateOrUpdate.json

```diff
@@ -0,0 +1,69 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testResourceGroup",
+    "connectionName": "testManagedApi",
+    "api-version": "2016-06-01",
+    "apiConnection": {
+      "properties": {
```

Comment:
most of these properties are read-only; on PUT one would only provide parameterValues, customParameterValues, DisplayName and API reference (just the id)

Reply:
Updated